pkg/audit/disqualification_test.go

```diff
@@ -0,0 +1,94 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package audit_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+	"go.uber.org/zap"
+	"storj.io/storj/internal/testcontext"
+	"storj.io/storj/internal/testplanet"
+	"storj.io/storj/pkg/audit"
+	"storj.io/storj/pkg/overlay"
+	"storj.io/storj/pkg/storj"
+	"storj.io/storj/satellite"
+)
+
+// TestDisqualificationTooManyFailedAudits does the following:
+// * Create a failed audit report for a storagenode
+// * Record the audit report several times and check that the node isn't
+//	 disqualified until the audit reputation reaches the cut-off value.
+func TestDisqualificationTooManyFailedAudits(t *testing.T) {
+	var auditDQCutOff float64
+
+	testplanet.Run(t, testplanet.Config{
+		SatelliteCount: 1, StorageNodeCount: 1, Reconfigure: testplanet.Reconfigure{
+			Satellite: func(log *zap.Logger, index int, config *satellite.Config) {
+				auditDQCutOff = config.Overlay.Node.AuditReputationDQ
```

Comment:
You don't really need this to be in the Reconfigure. You can get this value in the test body too.

Reply:
That's what I wanted but I didn't find how.

Could you tell me how to access to it please?

Reply:
Oh, I assumed wrong! I thought it is possible to access peer config in the test body, but this is not possible. Perhaps, this could be a nice future improvement for testplanet.

Reply:
I thought the same, I'll open a ticket for it

Reply:
https://storjlabs.atlassian.net/browse/V3-2011